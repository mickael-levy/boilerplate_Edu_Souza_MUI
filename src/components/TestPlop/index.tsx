import * as S from "./testPlop.styles";

function TestPlop() {
  return (
    <S.Wrapper>
      <h1>testPlop</h1>
    </S.Wrapper>
  );
}

export default TestPlop;
