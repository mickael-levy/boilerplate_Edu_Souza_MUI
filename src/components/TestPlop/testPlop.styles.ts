/* eslint-disable @typescript-eslint/no-unused-vars */
import styled, { css } from "styled-components";

export const Wrapper = styled.div`
  ${({ theme }) => css``}
`;
