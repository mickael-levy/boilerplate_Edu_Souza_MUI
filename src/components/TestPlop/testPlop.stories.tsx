import { Story, Meta } from "@storybook/react/types-6-0";
import TestPlop from ".";

export default {
  title: "TestPlop",
  component: TestPlop,
} as Meta;

export const Default: Story = () => <TestPlop />;
