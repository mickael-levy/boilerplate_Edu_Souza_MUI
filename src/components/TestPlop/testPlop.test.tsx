import { screen } from "@testing-library/react";
import { renderWithTheme } from "@/utils/tests/helpers";

import TestPlop from ".";

describe("<TestPlop />", () => {
  it("should render the heading", () => {
    const { container } = renderWithTheme(<TestPlop />);

    expect(
      screen.getByRole("heading", { name: /TestPlop/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
