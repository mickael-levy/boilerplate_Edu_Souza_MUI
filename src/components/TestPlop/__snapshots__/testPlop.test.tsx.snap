// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestPlop /> should render the heading 1`] = `
.c0 {
  background-color: #F231A5;
}

<div
  class="c0"
>
  <h1>
    testPlop
  </h1>
</div>
`;
